src/webots/app/WbPerspective.cpp

```diff
@@ -389,8 +389,6 @@ QString WbPerspective::getActionName(WbAction::WbActionKind action) {
       return "objectMoveDisabled";
     case WbAction::DISABLE_FORCE_AND_TORQUE:
       return "forceAndTorqueDisabled";
-    case WbAction::DISABLE_FAST_MODE:
```

Comment:
The `DISABLE_3D_VIEW` status should be added in the perspective file (as it was the case for the `DISABLE_FAST_MODE` option.

Reply:
This seems not to work correctly (initial state of `DISABLE_FAST_MODE` is not applied) in Webots 2020b Rev 1, maybe we should address that.

See: https://github.com/cyberbotics/webots/pull/2286/commits/ca8f681c6555aeb43cddb42e14796a8100b00644

Reply:
Yes, this should also be addressed on the master branch then.
But we should in any case write and read the new `DISABLE_3D_VIEW` options from the perspective file.
So it seems that the new option should be added here.